libs/cardano-ledger-api/test/Test/Cardano/Ledger/Api/Tx/Body.hs

```diff
@@ -89,10 +89,28 @@ evaluateTransactionBalance pp dpstate (UTxO u) txBody = consumed <-> produced
     refunds = keyTxRefunds pp dpstate txBody
     withdrawals = fold . unWithdrawals $ txBody ^. withdrawalsTxBodyL
 
+evaluateTransactionBalanceShelley ::
+  (EraTxBody era, ShelleyEraTxCert era) =>
+  PParams era ->
+  CertState era ->
+  UTxO era ->
+  TxBody era ->
+  Value era
+evaluateTransactionBalanceShelley pp dpstate (UTxO u) txBody = consumed <-> produced
```

Comment:
This duplication is a bit sad, couldn't we define the function  `evaluateTransactionBalance` above using this one: 
```
evaluateTransactionBalance pp dpstate utxo txBody =
    evaluateTransactionBalanceShelley pp dpstate utxo txBody<> (txBody ^. mintValueTxBodyF)
```
?